pkg/cmd/stack/deploy.go

```diff
@@ -555,6 +554,43 @@ func isAnyPortAvailable(ctx context.Context, svc *model.Service, stack *model.St
 	return false
 }
 
+func deployK8sService(ctx context.Context, svcName string, s *model.Stack, c kubernetes.Interface, spinner *utils.Spinner) error {
+	svcK8s := translateService(svcName, s)
+	old, err := services.Get(ctx, svcName, s.Namespace, c)
+	if err != nil {
+		if !oktetoErrors.IsNotFound(err) {
+			return fmt.Errorf("error getting ingress '%s': %w", svcName, err)
+		}
+		if err := services.Deploy(ctx, svcK8s, c); err != nil {
+			return err
+		}
+		spinner.Stop()
+		oktetoLog.Success("Endpoint '%s' created", svcName)
+		spinner.Start()
+		return nil
+	}
+
+	if old.GetLabels()[model.StackNameLabel] == "" {
+		oktetoLog.Warning("skipping deploy of %s due to name collision: the ingress '%s' was running before deploying your compose", svcName)
```

Comment:
I think the deploy could be from another place than the compose, right?

Reply:
No, We check that the new and the old services come from a compose service by checking their labels, so the user will only get the message when the run from a compose

Reply:
Yes, probably is not the right words but what we tried to explain to the user is that there is a k8s service already running with that name and we are not going to deploy the k8s service